docs/about/contributors.md

```diff
@@ -1,8 +1,8 @@
 # Contributors
 
-The Operational Data Standard Contributors Group (formerly known as the ODS Working Group) is convened by staff from the [California Integrated Travel Project](http://calitp.org) as part of its goal to provide complete, accurate, and up-to-date transit data to customers and respective agencies.
+The Transit Operational Data Standard Contributors Group (formerly known as the ODS Working Group) is convened by staff from the [California Integrated Travel Project](http://calitp.org) as part of its goal to provide complete, accurate, and up-to-date transit data to customers and respective agencies.
```

Comment:
Update to: The Transit Operational Data Standard Contributors Group (formerly known as the ODS Working Group) is convened by [TODS Board](../governance/governance.md#TODS-board-of-directors).

Reply:
Below we refer to the working group as the TODS Working Group and here we refer to it as the ODS Working Group. I'm fine with either way since it was referred to as the ODS Working Group at the time, but we should pick one and stick with it.

Reply:
fixed in https://github.com/cal-itp/operational-data-standard/pull/78/commits/425636d3e31d980d39fed61d128f1fcdc854250e